src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -298,6 +299,8 @@ def launch_fluent(
 
     Parameters
     ----------
+    product_version: str, optional
+        Sets the product version. Options are ``"22.2"`` and ``"23.1"``.
```

Comment:
@hpohekar From the code below, it seems to take strings `22.2.0` or `23.1.0` only. If we are adding a `product_version` argument, it should accept the version in the same fashion as in `set_ansys_version` for consistency.

Or, we can just call `set_ansys_version` before `launch_fluent` in `setup_for_fluent` without chaning `launch_fluent`'s signature.

Reply:
Right. We will update this now.

Reply:
Done.

Reply:
Both ```__main__.py and launch_fluent``` updated now.